gh-pages/_docs/05-06-docker-containers.md

```diff
@@ -9,18 +9,18 @@ docker-compose.yml
 
 ## The Container Landscape
 
-| Container-Name           | Description                             | How to use                                                      |
-| ------------------------ | --------------------------------------- | --------------------------------------------------------------- |
-| codecharta-sonar         | Hosts an instance of SonarQube          | localhost:9000 in the browser, follow the steps for Linux       |
-| codecharta-visualization | Runs the visualisation part o CodeChara | localhost:9001, load files from your hard-drive                 |
-| codecharta-analysis      | Contains all tools the ccsh can import  | Connect via terminal `docker exec -it codecharta-analysis bash` |
+| Container-Name           | Description                             | How to use                                                |
+| ------------------------ | --------------------------------------- | --------------------------------------------------------- |
+| sonar                    | Hosts an instance of SonarQube          | localhost:9000 in the browser, follow the steps for Linux |
+| codecharta-visualization | Runs the visualisation part o CodeChara | localhost:9001, load files from your hard-drive           |
+| codecharta-analysis      | Contains all tools the ccsh can import  | Connect via terminal `docker exec -it analysis bash`      |
```

Comment:
```suggestion
| codecharta-analysis      | Contains all tools the ccsh can import  | Connect via terminal `docker exec -it codecharta-analysis bash`      |
```
Please don't forget to re-add codecharta here, it's a bit hard to spot admittedly 😅